lib/nel.js

```diff
@@ -49,9 +49,27 @@ var fs = require("fs");
 var path = require("path");
 var spawn = require("child_process").spawn;
 
+function findFile(file, dir) {
+    var result = fs.readdirSync(dir).find(result => result.indexOf(file) >= 0);
```

Comment:
This isn't working because `result` is just the filename. How about this?

```js
    var found = ~fs.readdirSync(dir).indexOf(file);
    if (found) {
        return path.join(path.resolve(dir), file);
    }
```